9.2/root/usr/bin/run-postgresql

```diff
@@ -16,5 +16,10 @@ fi
 
 set_passwords
 
+if [ -f ${CONTAINER_SCRIPTS_PATH}/post-init.sh ]; then
```

Comment:
You may use `[ -x "${CONTAINER_SCRIPTS_PATH}/hooks/post-init" ]` to check that the file is executable?

And as a "bonus" in an else if clause check that it just exists (not executable file), show a warning...